pkg/ksync/spec_list.go

```diff
@@ -65,7 +70,8 @@ func DeserializeSpecList(s *pb.SpecList) (*SpecList, error) {
 	}, nil
 }
 
-func allSpecs() (map[string]*Spec, error) {
+// getDesiredSpecItems returns the latest spec configuration from the disk using viper
+func getDesiredSpecItems() (map[string]*Spec, error) {
```

Comment:
This wording actually seems confusing to me. This gets the list of _already configured_ specs, so it's not desired.

Reply:
This is getting the desired list of specs which has been set in the config file using `ksync create` or manually editing the file. This file is watched and the current list of specs in memory are updated using this. So it seemed like a desired specs which has not been applied and is waiting for some process to apply the change.
Also `allSpecs` and `newItems` was sounding more confusing.

Reply:
I don't disagree with you, but I think maybe it could be better:
```suggestion
func getExistingSpecItems() (map[string]*Spec, error) {
```
Maybe?

Reply:
Renamed to `getExistingSpecItems()` 👍